docker-compose.test.yml

```diff
@@ -123,13 +123,11 @@ services:
       retries: 3
 
   portal-db:
-    image: postgres:14.3
+    image: pocketfoundation/test-portal-postgres:latest
```

Comment:
This should use a specific version tag: otherwise we could break multiple repositories through a single docker image update.

Reply:
We need to use the latest version because use the latest version for PHD. It wouldn't be accurate otherwise